provider/terraform.rb

```diff
@@ -100,23 +100,44 @@ def force_new?(property, resource)
                              force_new?(property.parent, resource))))
     end
 
-    # Returns the property for a given Terraform field path (e.g.
+    # Returns tuples of (fieldName, list of update masks) for
+    #  top-level updatable fields. Schema path refers to a given Terraform
+    # field name (e.g. d.GetChange('fieldName)')
+    def get_property_update_masks_groups(properties, mask_prefix: '')
```

Comment:
Do we have any tests for this file? These methods feel like they should be unit tested.

Reply:
👍